docs/topics/mta-cli-args.adoc

```diff
@@ -0,0 +1,248 @@
+
+| -t, --target string
+| target technology to consider for analysis. To specify multiple targets, repeat the parameter: --target <target_1> --target <target_2> etc.
+|====
+
+[id="cli-input-file-type-arguments-global-flags_{context}"]
+=== Global flags
+
+.{ProductShortName} Analyze application
+[cols="40%,60%a",options="header",]
+|====
+
+|Flag
+|Description
+
+| --log-level uint32
+| log level (default 4)
+
+| --no-cleanup
+| do not cleanup temporary resources
+|====
+
+
+
+[id="cli-output-argument_{context}"]
+== Specifying the output directory
+
+Specify the path to the directory to output the report information generated by {ProductShortName}.
+
+.Usage
+[source,options="nowrap",subs="attributes+"]
+----
+--output <OUTPUT_REPORT_DIRECTORY>
+----
+
+* If omitted, the report will be generated in an `<INPUT_ARCHIVE_OR_DIRECTORY>.report` directory.
+* If the output directory exists and the `--overwrite` argument has not been provided, the command will return an error:
++
+[source,options="nowrap",subs="attributes+"]
+----
+Error: output dir //home/username/<OUTPUT_REPORT_DIRECTORY> already exists and --overwrite not set
+----
+
+However, if you specify the `--overwrite` argument, {ProductShortName} will proceed to delete and recreate the directory. See the description of this argument for more information.
+
+[id="cli-source-argument_{context}"]
+== Setting the source technology
+
+A space-delimited list of one or more source technologies, servers, platforms, or frameworks to migrate from. This argument, in conjunction with the `--target` argument, helps to determine which rulesets are used. Use the `--list-sources` argument to list all available sources.
+
+.Usage
+[source,options="nowrap",subs="attributes+"]
+----
+--source <SOURCE_1> <SOURCE_2>
+----
+
+The `--source` argument now provides version support, which follows the link:http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html[Maven version range syntax]. This instructs {ProductShortName} to only run the rulesets matching the specified versions. For example, `--source eap5`.
+
+[WARNING]
+====
+When migrating to JBoss EAP, be sure to specify the version, for example, `eap6`. Specifying only `eap` will run rulesets for all versions of JBoss EAP, including those not relevant to your migration path.
+
+See link:{ProductDocIntroToMTAGuideURL}/index#migration_paths_getting-started-guide[Supported migration paths] in _{IntroToMTABookName}_ for the appropriate JBoss EAP version.
+====
+
+[id="cli-target-argument_{context}"]
+== Setting the target technology
+
+A space-delimited list of one or more target technologies, servers, platforms, or frameworks to migrate to. This argument, in conjunction with the `--source` argument, helps to determine which rulesets are used. If you do not specify this option, you are prompted to select a target. Use the `--list-targets` argument to list all available targets.
```

Comment:
`you are prompted to select a target` 

This doesn't apply anymore, if the user doesn't provide a target, mta-cli will analyze the input anyway based on the source and discovery rules